app/services/create_sponsorship.rb

```diff
@@ -0,0 +1,34 @@
+class CreateSponsorship
+
+  attr_reader :sponsorship, :sponsor, :orphan
```

Comment:
The point of using `attr_reader` here is not to provide public access to the instance variables but to encapsulate access to them within the class itself. These accessors can - and perhaps should - be made private.